js/modules/k6/ws/ws.go

```diff
@@ -123,6 +129,26 @@ func (*WS) Connect(ctx context.Context, url string, args ...goja.Value) (*WSHTTP
 					continue
 				}
 				for _, key := range headersObj.Keys() {
+					if key == HeaderKeyCookie {
```

Comment:
HTTP header names are case-insensitive, so using `strings.ToLower()` here would be preferable.

Reply:
[strings.EqualFold](https://golang.org/pkg/strings/#EqualFold) is ... way faster :) 
```
package main

import (
        "strings"
        "testing"
)

func BenchmarkEq(b *testing.B) {
        for i := 0; i < b.N; i++ {
                _ = strings.EqualFold("Cookie", "cookie")
        }
}

func BenchmarkLower(b *testing.B) {
        for i := 0; i < b.N; i++ {
                _ = strings.ToLower("Cookie") == "cookie"
        }
}
```
```
BenchmarkEq-8           69264416                17.9 ns/op             0 B/op          0 allocs/op
BenchmarkLower-8        21890653                59.3 ns/op             8 B/op          1 allocs/op
```

```

Reply:
Fixed it